CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+## Supported Release 1.6.0
+### Summary
+
+A substantial release made to create a clean base from which Rubocop may be implemented.
```

Comment:
I would advise giving more information in the summary, even if you mention the features that have been added.